common/views/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -169,6 +185,35 @@ const SegmentedControl: FunctionComponent<Props> = ({
   const { isEnhanced } = useContext(AppContext);
   const [isActive, setIsActive] = useState(false);
 
+  function onClick(
+    e: MouseEvent<HTMLAnchorElement>,
+    item: SegmentedControlItem
+  ): boolean | undefined {
+    const url = e.currentTarget.href;
+    const isHash = url.startsWith('#');
+
+    trackGaEvent({
+      category: 'SegmentedControl',
+      action: 'select segment',
+      label: item.text,
+    });
+
+    setActiveId && setActiveId(item.id);
+    isEnhanced && setIsActive(false);
+
+    // TODO: Do we still need this?  The original comment ("Assume we want to")
+    // didn't provide much context, and it's not clear if/how this changes the
+    // behaviour of the page, or indeed if it ever fires.
+    //
+    // The only page where we use SegmentedControl with anchors in the URL is
+    // the What's On page, and testing locally the `isHash` value will never
+    // be true – the `href` of the item is the full URL.
```

Comment:
I'm ~95% sure we could safely delete this, but I don't want to introduce yet more changes today and potentially break something else.

I did track down the commit which added it, but it doesn't have much more context: https://github.com/wellcomecollection/wellcomecollection.org/pull/3509/commits/71fc1867dde5bd6ae54c290c7767daa9b45eabbf